bindings/rust/bench/src/harness.rs

```diff
@@ -64,6 +64,9 @@ pub trait TlsBenchHarness: Sized {
 
     /// Get whether or negotiated version is TLS1.3
     fn negotiated_tls13(&self) -> bool;
+
+    /// Transfer given data one-way between connections
```

Comment:
Based on my understanding of how this function works, it seems like you're sending some data and then reading some data. In the benchmarking code, it looks like you first get the client to call this function, and then the server. So what you're really testing is the throughput from client to server. I wonder if it would be better to split this into two, a send() and recv(), so it's a little more obvious what's going on.

Reply:
The function does the send and receive in one function call, but I agree it's unclear. For example, with `Mode::Client`, data is given to the client, the client writes the TLS records into the shared buffer, the server reads the TLS records, and finally the server writes the data back as plain-text, all in the same call. I'll separate out `send()` and `recv()` for the two connections and turn `transfer()` into a provided method that uses `send()` and `recv()`.

Reply:
Using `send()` and `recv()` is a little more verbose, but I think it could be worth the clarity.